src/main/webapp/js/finscholar/init.js

```diff
@@ -19,9 +19,12 @@ goog.module('finscholar');
 const GoogDom = goog.require('goog.dom');
 const {HomePageController} = goog.require('finscholar.homepagecontroller');
 
-const init = () => {
+const init = async () => {
```

Comment:
This method is temporarily set to async for testing.

Reply:
Wouldn't it need to stay async since rendering the pages is in itself asynchronous?

Reply:
Not sure, but wouldn't it need to stay async since rendering the pages is in itself asynchronous?

Reply:
The init function will be synchronous if the first page we load doesn't contain data from servlet. It depends on how far we can get on this project lol